tests/webapps/test-war-hello/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,3 @@
+runtime: custom
+vm: true
+module: hello
```

Comment:
updated in Issue #23 already - https://github.com/GoogleCloudPlatform/jetty-runtime/commit/d07985542e9166a8c060ed75296bf165dda1a796